proposals/0019/README.md

```diff
@@ -1,66 +1,58 @@
 ---
 id: 0019
 title: PBnJ.next
-status: ideation
+status: approved
 authors: Jacob Weinstock <jweinstock@equinix.com>
 ---
 
 ## Summary
 
-PBnJ is in need of some updating to increase the maintainability,
-robustness and extensibility of the service. Under the hood PBnJ
-currently relies on `ipmitool` and `racadm`. This limits its hardware
-agnostic functionality. Scaling the service is also a concern at the
-moment as there is an in-memory task service for some endpoints.
+PBnJ is a Restful API service that is tasked with interacting with Base Management Controllers (BMC). Among many other things, these BMC's can power a machine on and off and set the next boot device to PXE, BIOS, disk, etc. As part of a Tinkerbell workflow execution, the first step is
 
-This proposal is intended to provide an option for how to increase the
-maintainability, robustness and extensibility of the service.  
+> On the first boot, the Worker is PXE booted
+
+This is where PBnJ could be called to set the next boot device to PXE and then called again to reboot the machine. PBnJ currently support BMC's that work with the `ipmitool` or `racadm` tool. The following vendors are known to work: Supermicro, Dell, ASRockRack.
+
+Currently, there is no mechanism in Tinkerbell to hook into this functionality (unfortunately, that mechanism is not what this proposal is about). The PBnJ service is only provided as a convenience for End-Users to hook into. While PBnJ does not currently have any integration into the Tinkerbell stack, it is very likely that it will be integrated into tink-server and be used to provide additional functionality to workflows in the future.
```

Comment:
tink-server will not need any support for pbnj. All workflows will start with a couple of pbnj actions.